src/Commands/Env/ClearCacheCommand.php

```diff
@@ -27,18 +27,21 @@ class ClearCacheCommand extends TerminusCommand implements SiteAwareInterface
      * @aliases env:cc
      *
      * @param string $site_env Site & environment in the format `site-name.env`
+     * @param array $options
+     * @option bool $framework_cache [true|false] Clear the CMS cache
+     * @option bool $varnish_cache [true|false] Clear the edge cache
      *
      * @usage <site>.<env> Clears caches for <site>'s <env> environment.
      *
      * @throws \Pantheon\Terminus\Exceptions\TerminusProcessException
      * @throws \Pantheon\Terminus\Exceptions\TerminusException
      */
-    public function clearCache($site_env)
+    public function clearCache($site_env, $options = ['framework_cache' => true, 'varnish_cache' => false,])
```

Comment:
-1 on adding flags to the existing env:cc command. Note that it's not reliable to turn off the varnish cache here, as the Drupal hook will clear the varnish cache even if the ygg workflow does not.

The existing behavior may be inconsistent with WordPress, i.e. WordPress might not clear the varnish cache if the ygg workflow does not. Ideally we would fix WordPress, and hook its cache clear operation the same way we do for Drupal. Maybe having ygg clear the varnish cache invariantly here would be a reasonable workaround.